tasks/pipeline.py

```diff
@@ -891,7 +891,7 @@ def update_circleci_config(file_path, image_tag, test_version):
     """
     Override variables in .gitlab-ci.yml file
     """
-    image_name = "datadog/agent-buildimages-circleci-runner"
+    image_name = "gcr.io/datadoghq/agent-circleci-runner"
```

Comment:
This change is because we changed the registry we pull images from https://github.com/DataDog/datadog-agent/pull/20596

Reply:
nice catch!